lua/plenary/busted.lua

```diff
@@ -245,10 +231,40 @@ mod.run = function(file)
     print("FAILED TO LOAD FILE")
     print(color_string("red", msg))
     print(HEADER)
-    os.exit(2)
+    if is_headless then
```

Comment:
nice touch

Reply:
That also fixes a bug that `PlenaryBustedFile %` just closed neovim. But now it acts kinda weird in a different way :rofl:

Reply:
Oh I hated myself when I put PlenaryBustedFile by mistake. fixes part of #32